release-notes.hbs.md

```diff
@@ -339,3 +363,8 @@ When connecting to Google's GKE clusters, an error might appear with the text `W
 
 - Known issue 1
 - Known issue 2
+
+#### <a id="apps-plugin-known-issues"></a> Tanzu CLI - Apps plug-in
+
+- When `workload get` is used alongside `--no-color`, emojis are not removed from output.
+- For supply chain resources, output column is filled with `kind/name`, which is not useful to troubleshoot any problems or follow the workload flow in general since this is not reachable by Kubernetes. Output will be updated to provide the fully qualified resource name rather than `Kind`.
```

Comment:
I don't think we advertise this as a debugging mechanism. I don't think this is a known issue. @heyjcollins can you confirm?